Editor/Window/RegisterComputeInput.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using AmazonGameLift.Editor;
+using Editor.CoreAPI;
+using UnityEngine.UIElements;
+
+namespace Editor.Window
+{
+    public class RegisterComputeInput : StatefulInput
```

Comment:
Same comment for structuring this into its own corresponding uxml

Reply:
Separated out to different uxml